### DISCUSSION TITLE
bruh

### DISCUSSION THREAD

**Discussion by theatomicboy81** [UkrainianBanderasCat/Digital-Logic-Sim2#35] [General]
![Screenshot_8](https://user-images.githubusercontent.com/107222563/173717891-b501f5f6-ce78-4260-bd76-8a374ccab9b4.png)
the game neeeeeeeds to be more efficient 
if theres a way to give you the save and if you want it i will

**Comment by firecerne** [comment]
If you send us the folder "C:\Users\NAMEOFYOURUSER\AppData\LocalLow\Sebastian Lague _ Lime inc_\Digital Logic Sim\SaveData\Nameofyoursave" I will be able to acces it

**Reply by theatomicboy81** [reply]
K ty

**Reply by theatomicboy81** [reply]
I can do it Monday
